editor/docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+version: "3.9"
+services:
+  web:
+    build: .
+    ports:
+      - "3000:3000"
```

Comment:
Can you please add the command option here as well for running the container?

Reply:
It would be great if you can add example.env, which may contain BASE_URL for api call. 
And then you can add it into environment as well in same docker-compose file

Reply:
Thanks for suggestion. I added the command option with `npm start`. Let me know if anything else to be added.